src/commands/logic-app/openRunInMonitoringView.ts

```diff
@@ -0,0 +1,471 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from "vscode";
+import { AzureTreeDataProvider, IAzureNode } from "vscode-azureextensionui";
+import { localize } from "../../localize";
+import { LogicAppRunTreeItem } from "../../tree/logic-app/LogicAppRunTreeItem";
+import { getAuthorization } from "../../utils/authorizationUtils";
+
+export async function openRunInMonitoringView(tree: AzureTreeDataProvider, node?: IAzureNode): Promise<void> {
+    if (!node) {
+        node = await tree.showNodePicker(LogicAppRunTreeItem.contextValue);
+    }
+
+    const title = localize("azLogicApps.monitoringViewTitle", "Monitoring View");
+    const options: vscode.WebviewOptions & vscode.WebviewPanelOptions = {
+        enableScripts: true
+    };
+    const panel = vscode.window.createWebviewPanel("monitoringView", title, vscode.ViewColumn.Beside, options);
+    const authorization = await getAuthorization(node.credentials);
+    const runNode = node as IAzureNode<LogicAppRunTreeItem>;
+    const { id: runId, location, resourceGroupName, workflowId } = runNode.treeItem;
+    const { subscriptionId } = runNode;
+    panel.webview.html = getWebviewContent({ authorization, location, resourceGroupName, runId, subscriptionId, title, workflowId });
+}
+
+interface IGetWebviewContentOptions {
+    authorization: string;
+    location: string;
+    resourceGroupName: string;
+    runId: string;
+    subscriptionId: string;
+    title: string;
+    workflowId: string;
+}
+
+const version = "1.31119.1.53868315.181126-1351";
```

Comment:
Move this to a constant file? Or at the very least put it near the top?

Also could it be slightly more descriptive, what version is this of?

Reply:
The version of the designer used to render the monitoring view.

Maybe I should move this to its own module like I did for the read-only designer, i.e., monitoringViewUtils.ts.